tests/unit/engine.rs

```diff
@@ -237,9 +237,9 @@ mod tests {
             );
         }
         let expected_hash: u64 = if cfg!(feature = "css-validation") {
-            9439492009815519037
+            15545091389304905433
```

Comment:
We don't need to update `ADBLOCK_RUST_DAT_VERSION`, because the matching logic isn't changed. Tt's only a optimization how we store rules. The current .dat files can be used as is.